cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ProcessConfiguration.java

```diff
@@ -47,21 +55,31 @@ public void setMaxHeapMemorySize(int maxHeapMemorySize) {
         return this.jvmOptions;
     }
 
+    public Collection<String> getProcessParameters() {
+        return processParameters;
```

Comment:
this.

Reply:
Documentation? The JvmOptions also doesn't have any

Reply:
I don't think that the documentation is necessary in this case, but please add the `this.` accessor.

Reply:
fixed